docs/source/dev_guide.rst

```diff
@@ -79,8 +79,7 @@ Do the same as above using the ``Makefile``.
 Running notebooks tests
 -----------------------
 
-Assuming that the ``raven`` conda env has already been created and is up-to-date and
-raven-wps has been installed with ``make develop``::
+Assuming that the ``raven`` conda env has already been created and is up-to-date and `raven-wps`` has been installed with ``make develop``::
```

Comment:
Missing a ``` ` ``` before `raven-wps`

Highlighting is better if using the following instead of `::`
```rst
.. code-block:: shell
  
    [...code...]

```